examples/README.md

```diff
@@ -0,0 +1,122 @@
+
+  ```ruby
+  depends 'ilo'
+  ```
+
+  a. :pushpin: **Tip:** Check out the [Chef docs](https://docs.chef.io/config_rb_metadata.html) to see what all else you can put in this metadata.rb file.
+
+6. Now we can edit our first recipe and add some configuration. Open up `recipes/default.rb` and add:
+
+  ```ruby
+  # Replace these credentials with those of your iLO(s)
+  my_ilos = [
+    { host: 'ilo1.example.com', user: 'Admin', password: 'secret123', ssl_enabled: false },
+    { host: 'ilo2.example.com', user: 'Admin', password: 'secret456', ssl_enabled: false }
+  ]
+
+  ilo_user 'bob' do
+    ilos my_ilos
+    password 'password123'
+    login_priv true
+  end
+  ```
+
+  a. There's a few things I want to note here. The first thing is that the `my_ilos` object is just a Ruby array containing 2 Ruby hashes. These hashes contain the necessary data for connecting to the 2 iLOs we want to manage. We put this information in clear text here, but you'll definitely want to make some changes in production code, so you're not checking passwords into a repo in clear text. You can build this array by reading in a file, fetching an encrypted databag, etc. Also, instead of setting ssl_enabled to false, you should import the server's certs onto your machine.
+
+  b. The second thing I want to note is the ilo_users resource. Here we're just defining a user named bob, associating him with both of the ilos in our `my_ilos` array, and setting his password to `password123`. When run, this will create this user and set his password. You can run the recipe over and over, and it will continue to check if bob exists and set his password to what you've defined.
+
+7. Now let's run our recipe and see what happens:
+
+  ```bash
+  $ chef-client -z -o my_ilo::default
+  ```
+
+  a. You'll see an error saying it can't find the correct cookbook. This is because we haven't told Chef where to find our cookbooks, and we haven't downloaded the ilo cookbook. Fix this by creating a knife.rb file at `chef-repo/.chef/knife.rb` and adding:
+
+    ```ruby
+    # See http://docs.getchef.com/config_rb_knife.html for more information on knife configuration options
+
+    current_dir = File.dirname(__FILE__)
+    cookbook_path            ["#{current_dir}/../cookbooks"]
+
+    # If you're behind a proxy, you'll need to set the http_proxy environment variable or set the http_proxy option here
+    ```
+
+  b. And download the ilo cookbook by running:
+
+    ```bash
+    $ cd ..
+    # (Now we're in chef-repo/cookbooks)
+
+    $ knife cookbook site download ilo
+
+    # Now we need to untar the download:
+    $ tar -xvf ilo-*.tar.gz
+    # Now the ilo cookbook lives at chef-repo/cookbooks/ilo
+
+    # You can remove the tar file if you'd like; we no longer need it
+    $ rm ilo-*.tar.gz
+
+    # Do the same for the compat_resource cookbook:
+    $ knife cookbook site download compat_resource
+    $ tar -xvf compat_resource-*.tar.gz
+    $ rm compat_resource-*.tar.gz
```

Comment:
Just a suggestion. We can mention here why do we need compat_resource cookbook. And another way to manage this by using berkshelf(berks install)